src/Api/Service/Sf.php

```diff
@@ -657,7 +672,7 @@ public function lex_project_update($settings)
         return LexProjectCommands::updateProject($this->projectId, $this->userId, $settings);
     }
 
-    public function lex_baseViewDto()
```

Comment:
This seems like a random change. Especially since you haven't change the corresponding name for every front end angular service that uses it. BTW DTO stands for Data Transport Object; it means we have collected together items (potentially from different sources) usually for a particular page so that we only hit the server once.